src/modules/Invoice/Service.php

```diff
@@ -1128,15 +1128,15 @@ public function generateFundsInvoice(\Model_Client $client, $amount)
 
         $systemService = $this->di['mod_service']('system');
 
-        $min_amount = $systemService->getParamValue('funds_min_amount', null);
-        $max_amount = $systemService->getParamValue('funds_max_amount', null);
+        $min_amount = $systemService->getParamValue('minimum_funds_limit', null);
+        $max_amount = $systemService->getParamValue('maximum_funds_limit', null);
 
         if ($min_amount && $amount < $min_amount) {
-            throw new \Box_Exception('Amount is not valid', null, 981);
+            throw new \Box_Exception('Amount must be at least :min_amount', [':min_amount' => $min_amount], 981);
```

Comment:
We should look into making them translatable

Reply:
They are. Just need to update the sources once merged